installer/resources/pacbot_app/files/ui_docker/dockerfile

```diff
@@ -1,28 +1,37 @@
-FROM nginx:1.21.6
+# Use the official Nginx image with a specific version
+FROM nginx:1.25.3-alpine
 
 LABEL maintainer="NGINX Docker Maintainers <docker-maint@nginx.com>"
 
 ARG HTML_FILE
 ENV HTML_FILE=$HTML_FILE
 
 # Install required packages
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
+RUN apk update && \
+    apk upgrade  && \
+    apk add --no-cache \
         curl \
         python3 \
-        python3-distutils \
         unzip \
-        gnupg \
-    && rm -rf /var/lib/apt/lists/* \
-    && curl -O https://bootstrap.pypa.io/pip/3.6/get-pip.py \
-    && python3 get-pip.py \
-    && pip install awscli
+        gnupg
 
-# Create the target directory
+# Install pip using the recommended way for Alpine Linux
+RUN python3 -m ensurepip && \
+    rm -r /usr/lib/python*/ensurepip && \
+    pip3 install --upgrade pip setuptools
+
+# Install AWS CLI using pip
+RUN pip3 install awscli
```

Comment:
Why does the web app container need access to AWS cli?

Reply:
@plyubich 
We copy jar file from s3 to image. https://github.com/PaladinCloud/CE/blob/master/installer/resources/pacbot_app/files/ui_docker/entrypoint.sh.tpl

Reply:
can that jar be copied to docker beforehand like you do with Nginx config below?

Reply:
Webapp dist.zip folder is different for each Tenant's.
Configuration.ts file will vary for each Tenant's

Reply:
We need to move values from Tenant configuration value from configuration.ts to environmental variable